### PR TITLE
Simplify local configuration management

### PR DESCRIPTION
**Pull request by racodond**

### CODE REVIEW

File: bin/ntaf.js [hunk 1]
```diff
@@ -3,41 +3,38 @@
...
+let allPromises = [
+  fs.copy('node_modules/ntaf/template/', '.'),
+  fs.copy('node_modules/ntaf/Readme.md', 'Readme.md'),
+];
 
-} else if (process.argv[2] == 'run') {
-  console.log('run');
-} else {
-  console.log('Usage: ntaf install|run');
+for (let i in emptyDirectories) {
```
Comment (callmemagnus) [inline bin/ntaf.js]: emptyDirectoripoes is an array, prefer:

```
emptyDirectory.forEach(item => {
  ...
})
```

File: bin/ntaf.js [hunk 2]
```diff
@@ -3,41 +3,38 @@
...
+    (err) => onError(err)
+  );
+
 function onError(err) {
-  if (err) {
-    console.error(err);
-    process.exit(1);
-  }
+  console.error(err);
+  process.exit(1);
```
Comment (callmemagnus) [inline bin/ntaf.js]: Why exit 1 if no error.

Comment (racodond) [inline bin/ntaf.js]: This function is always called when there is an error.

File: bin/ntaf.js (hunk 1 above)
Comment (racodond) [inline bin/ntaf.js]: Thanks! Will take into account in the next commit.

Comment (racodond) [inline bin/ntaf.js]: https://github.com/nespresso/ntaf/commit/9b2e7453451ebb46d5cc143890000d4266a80b71

**Pull request merged**